homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,27 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
+    if (sequence == 1) return 1;
+    if (sequence == 2) return 1;
```

Comment:
Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`